pex/pex_bootstrapper.py

```diff
@@ -104,36 +115,51 @@ def maybe_reexec_pex(compatibility_constraints):
   :param compatibility_constraints: list of requirements-style strings that constrain the
```

Comment:
Outdated docstring

Reply:
Also why make this change? It doesn't appear that we use `pex_info` for anything other than `pex_info.interpreter_constraints`. By requiring `pex_info`, we make this function more complex and harder to test.

Reply:
Reverted and docstring updated. The pexrc verbage was always irrelevant and overly constraining.